### ISSUE TITLE
Proposal to add impacted service resource type as an additional information in the output formats supported by prowler

### ISSUE THREAD

**Issue by rams3sh** [prowler-cloud/prowler#578]
**What is the issue ?**
Currently I use prowler results (including historical results) of companies I consult in creating trends of compliance of their accounts. As of now , due to lack of information on services for the findings in prowler output, I use a fuzzy logic of terms to be present in a title to confirm the impacted resource type the finding corresponds to. This helps me creating a AWS resource type specific trend.

I consume csv export of prowler to create a trend and timeline chart.

**Current State ?**
Currently prowler gives the following info (taken from csv export of prowler scan) :-
```
PROFILE
ACCOUNT_NUM
REGION
TITLE_ID
RESULT
SCORED
LEVEL
TITLE_TEXT
NOTES
```
**Expected output**

I would like prowler to provide an additional field apart from the already existing fields i.e SERVICE_RESOURCE_TYPE.

_Example :_ 
i. For check `[check12] Ensure multi-factor authentication (MFA) is enabled for all IAM users that have a console password (Scored)`, it should have SERVICE_RESOURCE_TYPE as `USER`.

ii. For check `[check19] Ensure IAM password policy requires minimum length of 14 or greater (Scored)`, it should have SERVICE_RESOURCE_TYPE as `AWS ACCOUNT`.

iii. For check `[check122] Ensure IAM policies that allow full \"*:*\" administrative privileges are not created (Scored)`, it should have SERVICE_RESOURCE_TYPE as `POLICY`.

**Comment by toniblyx** [comment]
Hi @rams3sh, thanks for your comment. We are already adding ResourceType to some checks because is needed for ASFF, look at this:
https://github.com/toniblyx/prowler/blob/master/checks/check_extra730#L20

something like that is what you would like?

**Comment by toniblyx** [comment]
Ok I see this is related to https://github.com/toniblyx/prowler/issues/577#issuecomment-619602509

**Comment by toniblyx** [comment]
This has been already implemented using servicename in outputs since v2.5. ASFF doesn't include it OOTB but has added that attribute in the title.
